tests/boot/boot_from_pxe.pm

```diff
@@ -109,10 +119,25 @@ sub run {
     save_screenshot;
 
     if ((check_var('BACKEND', 'ipmi') && !get_var('AUTOYAST')) || get_var('SES5_DEPLOY')) {
-        my $ssh_vnc_wait_time = 600;
-        assert_screen((check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc') . '-server-started', $ssh_vnc_wait_time);
-        select_console 'installation';
+        my $ssh_vnc_wait_time = get_var('SES5_DEPLOY') ? 300 : 210;
+        my $ssh_vnc_tag = eval { check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc' } . '-server-started';
```

Comment:
why the eval?

Reply:
> why the eval?

I encountered an programming error during test run time. So adopted this way of usage. Any concern ?

Reply:
I don't see where the `check_var` should be necessary. Please remove the `eval` unless we have a good reason to keep it. We should not try to "safe-guard" every call to any function and `check_var` is just a function from the os-autoinst test API that is supposed to work. If it does not work correctly we should not try to hide these errors but be explicit about it